docs/contributing/tests.md

```diff
@@ -0,0 +1,55 @@
+# Running Tests
+
+> Tests ensure that the platform functions as expected when the existing codebase is modified. For more details on
+> writing of tests, please refer to the following links.
+
+- [App unit tests](/contributing/tests/app-unit-tests)
+- [API unit tests](/contributing/tests/api-unit-tests)
+- [Blackbox tests](/contributing/tests/blackbox-tests)
+
+## Running app unit tests
+
+Run `pnpm --filter app test`.
+
+## Running API unit tests
+
+Run `pnpm --filter api test`.
+
+## Running blackbox tests
+
+Install [Docker](https://docs.docker.com/get-docker/) and ensure that the service is running.
+
+Run the following from the project's root directory.
+
+```bash
+# Ensure that you are testing on the lastest codebase
+pnpm build
+
+# Clean up in case you ran the tests before
```

Comment:
Is this still recommended?

Reply:
Yes, as we could directly access the databases where some actions  / changes may affect the tests. Would you prefer to change this into a warning instead?

Reply:
Okay, thanks! No, this is fine for me then 👍